torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7572,6 +7572,27 @@ def sample_inputs_pdist(op_info, device, dtype, requires_grad, **kwargs):
         *[SampleInput(make_input((S, S)), kwargs=dict(p=p)) for p in (0.0, 1.0, 2.0, 10.0, float("inf"))],
     ]
 
+def sample_inputs_binary_cross_entropy(op_info, device, dtype, requires_grad, logits=False, **kwargs):
+    make = partial(make_tensor, device=device, dtype=dtype)
+    make_prob = partial(make, low=0, high=1)
+
+    reductions = ("mean", "sum", "none")
+
+    shapes_and_kwargs = [
```

Comment:
does this test the `pos_weight` arg of BINARY_CROSS_ENTROPY_WITH_LOGITS?

Reply:
Nope, I completely missed that, because it is not available for the op without logits. Good catch!